ci/linkcheck.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
```suggestion
#!/usr/bin/env bash

set -ev
set -o pipefail
```

Reply:
What do these do?

Reply:
`set -e` exits on error. `set -v` prints each command as it's executed, which makes it easier to debug. `pipefail` counts the command as failing if any command in pipeline fails; the default is it will have the status of the last command in the pipeline, which can ignore errors.